index.html

```diff
@@ -28,7 +28,136 @@ <h1>I’m Mariane Glad to see you!</h1>
         <li class="icons_list"><a href="#"><img src="./logos/Vector.png" alt="navigation icon"></a></li>
     </ul>
 </section>
+    <section class="cards">
+        <div class="card">
+            <img src="./images/SnapshootPortfolio.png" alt="navigation icon" class="snapshoot">
+             <h1>Tonic</h1>
+             
+             <div class="sec">
+                <p>CANOPY</p>
+                <span class="dot"></span>
+                <p class="sec_color1">Back End Dev</p>
+                <span class="dot"></span>
+                <p class="sec_color2">2015</p>
+            </div>
+            
+            <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="list">
+                <li class="box"><button type="button">html</button></li>
+                <li class="box"><button type="button">css</button></li>
+                <li class="box"><button type="button">javaScript</button></li>
+            </ul>
+            
+            <button class ="card_see" type="button">See Project</button>
```

Comment:
- [ ] Good job using a `button` tag for this but please kindly ensure that the style for its different states(hover, pressed and inactive) are taken care of.

![image](https://user-images.githubusercontent.com/43995830/164948400-1339280c-0159-4e34-81d8-1c297991bc91.png)